speculos/mcu/bagl.py

```diff
@@ -155,7 +162,7 @@ def hal_draw_rect(self, color, x, y, width, height):
                 break
 
     @staticmethod
-    def compute_line_width(font_id, width, text, text_encoding):
+    def _compute_line_width(font_id: int, width: int, text: bytes) -> int:
```

Comment:
Not related to this PR, but I'm not really sure why we need to perform line width computation ? It should be done by the device ?

Reply:
Ah ok this might be an MCU API actually, that's ugly to redo the code here, if it changes in the device we'll have real differences..

Reply:
Honestly, I have no clue!